CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Contribute to Flutter docs
+description: Ways to contribute to the Flutter project.
+---
+
+If you would like to help contribute to the Flutter project,
+we’re happy to have your help! Anyone can contribute, whether you’re new to
+the project or you’ve been around a long time, and whether you self-identify
+as a developer, an end user, or someone who just can’t stand seeing typos.
+
+For more ways to get involved in the Flutter community or to learn about us,
+also visit the [Flutter community](/community) page.
+
+Looking for the [Effective Dart]({{site.www}}/guides/language/effective-dart)?
```

Comment:
i don't really understand why this paragraph about effective dart is on this page. It seems weird to link to the Dart style guide but not, say, the Flutter style guide. (Not that either really make sense to link to from this doc as I see it, but maybe I'm missing something?)